source/jormungandr/jormungandr/scenarios/utils.py

```diff
@@ -461,3 +462,62 @@ def include_poi_access_points(request, pt_object, mode):
         ]
         and pt_object.poi.children
     )
+
+
+def get_impact_uris_for_poi(response, poi_uri):
+    impact_uris = []
+    for impact in response.impacts:
+        for object in impact.impacted_objects:
+            if object.pt_object.embedded_type == type_pb2.POI and object.pt_object.uri == poi_uri:
+                impact_uris.append(impact.uri)
+
+    return impact_uris
+
+
+def fill_disruptions_on_pois(instance, response):
+    if len(response.pois)  > 0:
+        pt_planner = instance.get_pt_planner("loki")
```

Comment:
I think this will raise an exception if loki is not configured for this instance

Reply:
:+1: